stripe/src/test/java/com/stripe/android/util/StripeTextUtilsTest.java

```diff
@@ -245,4 +244,183 @@ public void asFundingType_whenUnknown_returnsUnknown() {
     public void asFundingType_whenGobbledegook_returnsUnkown() {
         assertEquals(Card.FUNDING_UNKNOWN, StripeTextUtils.asFundingType("personal iou"));
     }
+
+    @Test
```

Comment:
Sure -- the reason I didn't do all_the_brands is that there are two schemes for separation, "Amex", and "all others." I can add one for unknown too, since it's a special case.